tests/app_test.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from pidgin.app import *
+
+def test_translate_dict_to_json():
+    input = {"key1": "value1", "key2": "value2", "key3": {"key4": "value4"}}
+    output = translate_dict_to_json(input)
+    expected = '{"key1": "value1", "key2": "value2", "key3": {"key4": "value4"}}'
+    assert output == expected
+
+def test_translate_dict_to_bibtex():
```

Comment:
As a general note, for tests it's helpful to include a docstring that explains what the test is checking for, and why it might fail or what went wrong if it failed. These tests are pretty straightforward, but just something to keep in mind